python/TestHarness/testers/Tester.py

```diff
@@ -41,6 +41,8 @@ def validParams():
     params.addParam('petsc_debug',   ['ALL'], "{False,True} -> test only runs when PETSc is configured with --with-debugging={0,1}, otherwise test always runs.")
     params.addParam('curl',          ['ALL'], "A test that runs only if CURL is detected ('ALL', 'TRUE', 'FALSE')")
     params.addParam('tbb',           ['ALL'], "A test that runs only if TBB is available ('ALL', 'TRUE', 'FALSE')")
+    params.addParam('pthreads',      ['ALL'], "A test that runs only if PTHREADS is available ('ALL', 'TRUE', 'FALSE')")
+    params.addParam('openmp',        ['ALL'], "A test that runs only if OPENMP is available ('ALL', 'TRUE', 'FALSE')")
```

Comment:
Fine by me, I can disable threading for these tests if that is what you
prefer.

On Tue, Mar 1, 2016 at 8:42 AM, Cody Permann <notifications@github.com>
wrote:

> In python/TestHarness/testers/Tester.py
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_idaholab_moose_pull_6375-23discussion-5Fr54584964&d=BQMCaQ&c=54IZrppPQZKX9mLzcGdPfFD1hxrcB__aEkJFOKJFd00&r=h7heP8xwI1i_HikChvhFbEBurKirgfOCdwgBxB9lM8c&m=e86LH6Ovf1SgYt521bchQq3CrvSsXgHN_9Q0mC-3gv8&s=rnd5_RCGyz4PnBvjNN5fKHgfm5II8Tn7FQXodIIQ9AY&e=>
> :
>
> > @@ -41,6 +41,8 @@ def validParams():
> >      params.addParam('petsc_debug',   ['ALL'], "{False,True} -> test only runs when PETSc is configured with --with-debugging={0,1}, otherwise test always runs.")
> >      params.addParam('curl',          ['ALL'], "A test that runs only if CURL is detected ('ALL', 'TRUE', 'FALSE')")
> >      params.addParam('tbb',           ['ALL'], "A test that runs only if TBB is available ('ALL', 'TRUE', 'FALSE')")
> > +    params.addParam('pthreads',      ['ALL'], "A test that runs only if PTHREADS is available ('ALL', 'TRUE', 'FALSE')")
> > +    params.addParam('openmp',        ['ALL'], "A test that runs only if OPENMP is available ('ALL', 'TRUE', 'FALSE')")
>
> I agree, I'd rather not have any of these flags either. As we already
> established they would most likely be used incorrectly [image: :smile:].
> If you really need to count something, let's just disable threading for
> those tests.
>
> —
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_idaholab_moose_pull_6375_files-23r54584964&d=BQMCaQ&c=54IZrppPQZKX9mLzcGdPfFD1hxrcB__aEkJFOKJFd00&r=h7heP8xwI1i_HikChvhFbEBurKirgfOCdwgBxB9lM8c&m=e86LH6Ovf1SgYt521bchQq3CrvSsXgHN_9Q0mC-3gv8&s=UdyRAMRHTakZ6MQEbemF7WV8_7ZSs59w2VuKUd8Eh5c&e=>
> .
>